src/components/controls/TextArea/TextArea.scss

```diff
@@ -46,31 +46,6 @@ $block: '.#{variables.$ns-new}text-area';
         }
     }
 
-    &__outer-additional-content {
```

Comment:
Shouldn't we also remove styles from 'TextInput.scss'?

Reply:
Of course. Missed